WTWSMS/events/VIET_Jewish_Flavor_Events.txt

```diff
@@ -599,22 +639,22 @@ character_event = {
 		}
 		any_courtier = {
 			limit = {
-				religion_group = jewish_group
-				culture_group = israelite
+				OR = {
+					religion_group = jewish_group
+					religion = ebionites
+				}
+#				culture_group = israelite
 			}
+
 			opinion = {
 				modifier = opinion_pleased
 				who = ROOT
 				years = 1
 			}			
 		}
-		random_list = {
-			20 = { }
-			80 = {
-				remove_trait = zealous
-			}
-		}
-		
+
+		remove_trait = zealous
+		add_trait = sympathy_judaism
```

Comment:
"Jews are the Chosen People eveyrone is child of God 'kumbyah'" is the comment behind it. Clearly, if a Kind ruler takes that Event, he sympathizes and understands Jews.